packages/manager/src/components/ImageSelectv2/ImageSelectv2.tsx

```diff
@@ -38,15 +46,23 @@ interface Props
   value: null | string | undefined;
   /**
    * Determines what images are fetched and shown
-   * - Public - Includes all public Linux distributions
+   * - Public - Includes all public Linux OSs
```

Comment:
acronyms ending in "S" tend to read a little awkwardly

```suggestion
   * - Public - Includes all public Linux OS's
```

or 

```suggestion
   * - Public - Includes all public Linux operating systems
```

Reply:
Reverted back to distribution since this is similar to the above comment